### DISCUSSION TITLE
have you checked their approach ?

### DISCUSSION THREAD

**Discussion by patientx** [blepping/comfyui_jankhidiffusion#7] [Ideas]
https://github.com/vladmandic/automatic/blob/dev/modules/hidiffusion/hidiffusion.py , they aren't using xformers as the official page or other custom nodes do but it seem to work still , if we can do this in comfy it would be cool.

**Comment by blepping** [comment]
the reason it works there is because the original HiDiffusion implementation is for Diffusers and vladmatic's A1111 fork has integration with Diffusers. ComfyUI has no Diffusers support, so the only way to use the original implementation is to just have a node that wraps diffusers like https://github.com/florestefano1975/ComfyUI-HiDiffusion

however there is a big disadvantage to that approach: you can only interact with it by feeding it a prompt and getting an image out, you can't do stuff like use ComfyUI model patches, LoRAs, samplers, etc.
